services/dask-sidecar/src/simcore_service_dask_sidecar/cli.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+import typer
+from settings_library.utils_cli import create_settings_command, create_version_callback
+
+from ._meta import PROJECT_NAME, __version__
+from .settings import Settings
+
+# SEE setup entrypoint 'simcore_service_dask_sidecar.cli:the_app'
+log = logging.getLogger(__name__)
```

Comment:
should we not go for, or is it not the case here?
```suggestion
_logger = logging.getLogger(__name__)
```